src/controller.cpp

```diff
@@ -46,6 +46,7 @@ Date: August 2020
 
 #include "hardware_definition.h"
 #include "utils.h"
+#include <time.h>
```

Comment:
I am not sure that the functionality of the time.h library is implemented on the Teensy uC. Use instead de millis() function to achieve the same functionality.